RELEASE-NOTES.md

```diff
@@ -22,6 +22,9 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
   ``BLT_REQUIRED_ASTYLE_VERSION``, ``BLT_REQUIRED_CLANGFORMAT_VERSION``, and ``BLT_REQUIRED_UNCRUSTIFY_VERSION``
 - Added ``HEADERS`` to ``blt_add_executable``.  This is important for build system dependency tracking
   and IDE folder support.
+- Added new ``blt_import_library`` macro that creates a real CMake target for imported libraries,
+  intended to be used instead of ``blt_register_library`` whenever possible
+- Added new ``blt_patch_target`` macro for modifying the properties of a CMake target in bulk
```

Comment:
For the second one "in bulk" seems a bit vague.

Perhaps something like the following would be clearer:
>Added new ``blt_patch_target`` macro to simplify modifying properties of an existing CMake target. This macro accounts for known differences in compilers and CMake releases.

Reply:
I think it also makes sense to mention that it takes into account differences in target types as some operations are only valid on certain types of CMake targets.